chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -46,4 +46,32 @@ spec:
         url: {{ .Values.global.tenantMappingService.url }}
   - handler: id_token
     config:
-      claims: {{ .Values.oathkeeper.idTokenConfig.claims | quote }}
\ No newline at end of file
+      claims: {{ .Values.oathkeeper.idTokenConfig.claims | quote }}
+---
+apiVersion: oathkeeper.ory.sh/v1alpha1
+kind: Rule
+metadata:
+  name: compass-gateway-certs
+spec:
+  description: Configuration of oathkeeper for secure endpoint of compass gateway
+  upstream:
+    url: http://compass-gateway.{{ .Release.Namespace }}.svc.cluster.local:3000
+  match:
```

Comment:
Could you please move there also a rule for Connector? If you don't want to merge these two rules into one for Gateway, then at least please keep it in one file. Thanks!